src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -701,7 +702,7 @@ public static string GetXmlSerializerAssemblyName(Type type)
             return GetXmlSerializerAssemblyName(type, null);
         }
 
-        public static string GetXmlSerializerAssemblyName(Type type, string defaultNamespace)
+        public static string GetXmlSerializerAssemblyName(Type type, string? defaultNamespace)
         {
             if (type == null)
             {
```

Comment:
on line 714: can events be nullable? cc @safern

Reply:
not sure, I think `Handler += null` would be a bit weird though 😄

Reply:
The concern is not so much about the `add` and `remove` methods but because the underlying `XmlNodeEventHandler` can be `null`, but I think that `_events.OnUnknownNode` nullability doesn't have to reflect on the public event so this is fine as is.